sentry/src/main/java/io/sentry/NoOpSpan.java

```diff
@@ -57,6 +57,9 @@ public void finish() {}
   @Override
   public void finish(@Nullable SpanStatus status) {}
 
+  @Override
+  public void finish(@Nullable SpanStatus status, @Nullable Date timestamp) {}
```

Comment:
l: Do we need a changelog entry for the new overload?

Reply:
I can add one but I don't think we usually mention every API change explicitly when implementing new features. I was planning on adding a single changelog entry for the whole basic OTEL support when merging #2344